x-pack/plugins/ml/public/application/explorer/explorer.js

```diff
@@ -328,6 +328,10 @@ export class ExplorerUI extends React.Component {
                     id="xpack.ml.explorer.topInfuencersTitle"
                     defaultMessage="Top influencers"
                   />
+                  <EuiIconTip
+                    content="This list shows the relative impact of the top influencers on the anomalies in the selected time period. Each influencer has a maximum anomaly score between 0-100 and a total anomaly score for that period."
```

Comment:
What about starting the tooltip with a verb? 

"Shows the relative impact on the ...."

Reply:
Thanks @gchaps.  Since there's an additional "add/remove filter" option in that section, I was actually pondering changing it like this:

> View the relative impact of top influencers on the anomalies in the select time period and add them as filters on the results....

Reply:
I've made this change in d4ed41dd18e1329fd7e0d8ec615f2c090004738e

Reply:
Nice change. "View" is better than "show" because it engages the user.